app/assets/javascripts/app/notifications/notifications.js

```diff
@@ -0,0 +1,18 @@
+(function() {
+  'user strict';
+
+  angular
+    .module('council.notifications')
+    .controller('NotificationsCtrl', NotificationsCtrl);
+
+  function NotificationsCtrl($state, Notification) {
+    var ctrl = this;
+
+    Notification.findAll()
+      .then(setNotifications);
+
+    function setNotifications(notifications) {
```

Comment:
Let's not use/maintain state in the controller. In this particular case you're just reassigning the resolved data intro a controller property. You could easily need to do some massaging with the resolved data, which would make you end up with unrelated logic inside you controller.
So as a rule of thumb, we don't maintain state in the controller, we actually resolve the data inside the service/factory itself, where it will manage it however it wants and then the controller simply requests the data when it's ready.
So, something like this:
```
Notification.findAll()
  .then(setNotifications);

function setNotifications() {
  ctrl.notifications = Notification.getAll();
}
```